docs/designs/flatten_odps_key_value_table.md

```diff
@@ -0,0 +1,107 @@
+# Transform the Column with Key-value Pairs String to Wide Table
+This document describes the design to transform the column with key-value pairs string to a wide ODPS table where
```

Comment:
This doc is a detailed design from the Data Analysis and Transform. How about add a link(https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/data_transform.md#normalize-table-schema-to-be-wide) to connect these two documents.

Reply:
OK，Good to me!